### ISSUE TITLE
GitHub plugin doesn't handle dotted-decimal IPv6 address

### ISSUE THREAD

**Issue by Mikaela** [ProgVal/Supybot-plugins#270]
Solution: implement https://github.com/ProgVal/Supybot-plugins/issues/230 ?

```
WARNING 2015-03-04T19:39:12 supybot '::ffff:192.30.252.46' tried to act as a web hook for Github,
            but is not GitHub.
```
https://github.com/ProgVal/Supybot-plugins/issues/168 would also be nice, but it's unreleated. 

This is caused by the HTTPd listening on `::0` which by default on most of Linux distributions makes it to bind to both alI IPv4 addresses and IPv6 addresses.

This is probably Limnoria bug and can be fixed with `net.ipv6.bindv6only=1` in sysctl options which again requires the bot to bind both IPv4 and IPv6 interfaces.

**Comment by Mikaela** [comment]
Releated options:

```
2015-03-04 22:43:37+0200 < Mikaela> ;;config help servers.http.hosts4
2015-03-04 22:43:37+0200 < Mikaela> ;;config help servers.http.hosts6
2015-03-04 22:43:37+0200 <@Limnoria> Space-separated list of IPv4 hosts the HTTP server will bind.  (Current value: 0.0.0.0)
2015-03-04 22:43:39+0200 <@Limnoria> Space-separated list of IPv6 hosts the HTTP server will bind.  (Current value: )
```

As GitHub is IPv4-only this can be fixed by setting the first as `0.0.0.0` and the latter to `""`/empty. This again disabled the web interface for all IPv6 users. If the previously mentioned systemctl option is set the options should be set as `0.0.0.0` and `::`.

**Comment by ProgVal** [comment]
Fixed by #286.

**Comment by nyuszika7h** [comment]
That's a workaround. I'm going to PR a proper fix to Limnoria.
